ios/ReactNativeCameraKit/RealCamera.swift

```diff
@@ -27,6 +27,7 @@ class RealCamera: NSObject, CameraProtocol, AVCaptureMetadataOutputObjectsDelega
     private let photoOutput = AVCapturePhotoOutput()
     private let metadataOutput = AVCaptureMetadataOutput()
 
+    private var resizeMode: ResizeMode = .contain
```

Comment:
Why do we do `.contain` here, and `.cover` in CameraView? Seems like we should have `contain` for all default values to keep things consistent (and say so in the doc)